tajo-cli/src/main/java/org/apache/tajo/cli/tsql/TajoCli.java

```diff
@@ -619,6 +623,10 @@ private void waitForQueryCompleted(QueryId queryId) {
         }
       }
 
+      if (isProgressPrinting == true) {
+        displayFormatter.printProgress(sout, status);
+        sout.println();
```

Comment:
Here, you added a new line. What does this new line mean? Please add some comments on that.